examples/python_boids_spatial3D_wrapped_native/boids_spatial3D.py

```diff
@@ -1,13 +1,12 @@
-!# /usr/bin/env python3
```

Comment:
Why have you removed the shebang?. This means you can't invoke the py file directly, `./boids_spatial3D.py`, but have to call it via `python3 boids_spatial3D.py`.

Reply:
Because it stops the python file being executed in my windows Python environment using `python file.py`.

Reply:
Fixed the typo and added back in both Python examples